redfish-core/lib/processor.hpp

```diff
@@ -2029,10 +2032,15 @@ inline void
         return;
     }
 
-    if (enabled.has_value())
-    {
-        patchCpuCoreMemberEnabled(asyncResp, coreId, *enabled);
-    }
+    auto callback = [asyncResp, coreId, enabled](const std::string& cpuPath) {
+        // Handle patched Enabled Redfish property
+        if (enabled.has_value())
+        {
+            patchCpuCoreMemberEnabled(asyncResp, cpuPath, coreId, *enabled);
+        }
+    };
+
+    getProcessorPaths(asyncResp, processorId, std::move(callback));
```

Comment:
As lambda body size is small enough, can it be inlined?

```
getProcessorPaths(asyncResp, processorId, 
           [asyncResp, coreId, enabled](const std::string& cpuPath) {
               // Handle patched Enabled Redfish property
               if (enabled.has_value())
               {
                   patchCpuCoreMemberEnabled(asyncResp, cpuPath, coreId, *enabled);
               }
           });

```

Reply:
Will address this as part of refactoring after the cherry-pick is completed